### PR TITLE
Sharing feature

### PR DESCRIPTION
**Pull request by lalaAdasia00**
Screenshot 2020-07-20 at 1 54 18 PM - Display 2
Screenshot 2020-07-20 at 1 54 26 PM - Display 2

### CODE REVIEW

File: scripts/share.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+var a2a_config = a2a_config || {};
```
Comment (nicholashusin) [inline scripts/share.js]: I think it would be best if we enforce camelCase usage across our JavaScript files.

File: stylesheets/user.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/*body {
```
Comment (nicholashusin) [inline stylesheets/user.css]: Any reason for committing a fully commented out file?

File: scripts/share.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+// A custom "onShare" handler for AddToAny
+function my_addtoany_onshare(data) {
+    events_demo.innerHTML = 'Shared &quot;<a href="'
+        + data.url
+        + '">'
+        + data.title
+        + '</a>&quot; to '
+        + data.service
+        + '!';
+}
```
Comment (ifeomi) [inline scripts/share.js]: what are these onReady and onShare handlers doing here?

**Comment by ifeomi** [comment]
this is really cool!!

File: stylesheets/user.css (hunk 2 above)
Comment (lalaAdasia00) [reply stylesheets/user.css]: No I was supposed to delete this. Sorry

File: scripts/share.js (hunk 3 above)
Comment (lalaAdasia00) [reply scripts/share.js]: Share Event Handling & Modifying but it is not needed anymore

File: html/user.hbs [hunk 4]
```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="/stylesheets/user.css">
```
Comment (ifeomi) [inline html/user.hbs]: i don't think this stylesheet is defined

**Review by ifeomi — APPROVED** [review]

File: scripts/share.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+var a2aConfig = a2aConfig || {};
```
Comment (nicholashusin) [inline scripts/share.js]: Probably want to use `let` instead of `var`

File: scripts/share.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+a2aConfig.templates = a2aConfig.templates || {};
+
+a2aConfig.templates.facebook = {
+    app_id: "5303202981",
```
Comment (nicholashusin) [inline scripts/share.js]: We should change the `"` to `'` in the `js` files.

**Review by nicholashusin — APPROVED** [review]
LGTM!
Looks really cool, thanks for adding this :)

**Review by chronologos — APPROVED** [review]

**Pull request merged**